File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/sporbarhetslog/Loggevent.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.sporbarhetslog;

import lombok.AllArgsConstructor;
import lombok.Data;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller.Orgnr;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller.InnloggetBruker;

@AllArgsConstructor
@Data
public class Loggevent {
  private InnloggetBruker innloggetBruker;
  private Orgnr orgnr;
  private boolean harTilgang;
  private String requestMethod;
  private String requestUrl;
  private String altinnServiceCode;
  private String altinnServiceEdition;
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/sporbarhetslog/Loggevent.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.sporbarhetslog

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.tilgangsstyring.InnloggetBruker
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene.Orgnr

data class Loggevent(
    val innloggetBruker: InnloggetBruker? = null,
    val orgnr: Orgnr? = null,
    val harTilgang: Boolean = false,
    val requestMethod: String? = null,
    val requestUrl: String? = null,
    val altinnServiceCode: String? = null,
    val altinnServiceEdition: String? = null,
)